server/test/e2e/setup.ts

```diff
@@ -12,10 +8,10 @@ export default async () => {
     .withReuse()
     .start();
 
+  process.env.NODE_ENV = 'development';
+  process.env.TYPESENSE_ENABLED = 'false';
+  process.env.IMMICH_MACHINE_LEARNING_ENABLED = 'false';
+  process.env.IMMICH_TEST_ENV = 'true';
   process.env.DB_URL = pg.getConnectionUri();
-
-  const redis = await new GenericContainer('redis').withExposedPorts(6379).withReuse().start();
-
-  process.env.REDIS_PORT = String(redis.getMappedPort(6379));
-  process.env.REDIS_HOSTNAME = redis.getHost();
+  process.env.TZ = 'Z';
```

Comment:
What does this do?

Reply:
TZ=Z sets timezone to UTC. Otherwise, the date returned by exiftool-vendored would be in the local timezone which causes failing tests